Update README.md (#141)
gravitee-io/gravitee-cli: README.md

## Gravitee CLI Documentation

> Gravitee cli documentation has been deprecated on https://docs.gravitee.io and will be migrated here soon.

Author: ```suggestion
The Gravitee CLI documentation is available [here](/docs) has been deprecated on https://docs.gravitee.io.
```
